plugins/star-rating/frontend/public/localization/star-rating.properties

```diff
@@ -21,7 +21,7 @@ star.number-of-ratings=NUMBER OF RATINGS
 star.percentage=PERCENTAGE
 feedback.popup-header-text = What's your opinion about this page?
 feedback.popup-comment-callout = Add comment
-feedback.popup-email-callaout = Contact me with e-mail
+feedback.popup-email-callout = Contact me with e-mail
```

Comment:
I see this fixed, but don't see fixes of it where it was used, just making sure that usage of the string was also fixed.